test/test_runner.py

```diff
@@ -0,0 +1,36 @@
+"""Validate utils from text module."""
+from subprocess import CompletedProcess
+
+from ansiblelint.runner import Runner
+
+SYNTAX_CHECK_STDERR = """\
+[WARNING]: No inventory was parsed, only implicit localhost is available
+[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
+ERROR! conflicting action statements: debug, always_run
+
+The error appears to be in '/foo/example.yml': line 47, column 7, but may
+be elsewhere in the file depending on the exact syntax problem.
+
+The offending line appears to be:
+
+
+    - name: always run
+      ^ here
+"""  # noqa: E501
```

Comment:
This should be part of the test case since nothing else uses it.

Reply:
While this sounds reasonable, there's a reason to keep it here: it's harder to preserve internal string nesting in nested code blocks. So keeping it outside improves maintainability.